src/rules/style.ts

```diff
@@ -39,8 +43,12 @@ export default {
   'new-parens': true,
   // Requires the use of `as Type` for type assertions instead of `<Type>`.
   'no-angle-bracket-type-assertion': true,
+  // Warns on comparison to a boolean literal, as in x === true.
+  'no-boolean-literal-compare': true,
   // Disallows one or more blank lines in a row.
   'no-consecutive-blank-lines': [true, 2],
+  // Disallow irregular whitespace outside of strings and comments
+  'no-irregular-whitespace': true,
```

Comment:
👍  (even though I secretly love column alignment in personal projects 😛)

[Examples](https://github.com/palantir/tslint/blob/master/test/rules/no-irregular-whitespace/test.ts.lint).